packages/salesforcedx-vscode-apex/src/index.ts

```diff
@@ -121,7 +121,7 @@ export async function activate(context: vscode.ExtensionContext) {
   context.subscriptions.push(commands);
 
   context.subscriptions.push(await registerTestView(testOutlineProvider));
-  const languageClientStatus = languageClientUtils.getStatus();
+  const languageClientStatus = languageClientUtils;
```

Comment:
Publishing `languageClientUtils.getStatus()` would not allow the consumers of this api to get the latest server status. Instead, they would get the status of the server at the point in which the apex extension got activated. Most of the time the status would be `Indexing`.

Reply:
I'd either change the name of the export from languageClientStatus to languageClientUtils, or export just getStatus with `const languageClientStatus = languageClientUtil.getStatus`.

Reply:
So `getStatus()` is instead called by whatever is consuming `languageClientStatus`? ie. `debugConfigurationProvider.ts` to get the status at that point of time?

Reply:
[nitpick] I agree with Bryan on the name.

Reply:
Cool, yes I'll send a follow up PR to clean up the naming.

Reply:
@AnanyaJha yes